src/leases/components/search/Search.js

```diff
@@ -236,6 +236,27 @@ class Search extends PureComponent<Props, State> {
             />
           </Column>
         </Row>
+        <SearchRow>
+          <SearchLabelColumn>
+            <SearchLabel>Palvelukokonaisuus</SearchLabel>
```

Comment:
Could this be something like `Suodata palvelukokonaisuuden mukaan`, to distinguish from the top nav bar filter. Should this label be put on a separate line above the field, just like in other fields?

You could also add a visible label for the search field above in the same way with text something like `Hae hakusanalla` or something else.

Reply:
Headerissa voisi lukea **Oma palvelukokonaisuus**
Suodatuksessa voisi lukea **Vuokrauspalvelu**